integration-test/background/storage.js

```diff
@@ -19,73 +17,111 @@ describe(`On https://${testPageDomain}/privacy-protections/storage-blocking/`, (
             },
             { polling: 100, timeout: 10000 }
         )
+        return { browser, page }
+    }
+
+    describe(`On https://${testPageDomain}/privacy-protections/storage-blocking/`, () => {
+        let cookies = []
 
-        let iframeFullyLoaded = false
-        try {
-            page.on('requestfinished', (req) => {
-                // once we see this url, we can consider the test completed
-                if (req.url().startsWith(`https://${thirdPartyTracker}/set-cookie`)) {
-                    iframeFullyLoaded = true
+        beforeAll(async () => {
+            let iframeFullyLoaded = false
+            const { browser, page } = await setup()
+            try {
+                page.on('requestfinished', (req) => {
+                    // once we see this url, we can consider the test completed
+                    if (req.url().startsWith(`https://${thirdPartyTracker}/set-cookie`)) {
+                        iframeFullyLoaded = true
+                    }
+                })
+                // Load the test pages home first to give some time for the extension background to start
+                // and register the content-script-message handler
+                await page.goto(`https://${testPageDomain}/`, { waitUntil: 'networkidle0' })
+                await page.bringToFront()
```

Comment:
For me this resolved issues when running the test locally, and was also convenient to see the test progress rather than just staring at the postinstall page until the tests failed :)